uniflow-android-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt

```diff
@@ -21,7 +20,7 @@ class SimpleObserver<T>(val callback: (T) -> Unit) : Observer<T> {
 class TestViewObserver {
     val values = arrayListOf<UIData>()
     val states = SimpleObserver<UIState> { values.add(it) }
-    val events = SimpleObserver<Event<UIEvent>> { values.add(it.peek()) }
+    val events = SimpleObserver<UIEvent> { values.add(it) }
```

Comment:
@arnaudgiuliani WDYT about observing `UIEvent` instead of `Event<UIEvent>` in tests?

Reply:
can be a good thing, yes 👍 

Event is just a wrapper for Android/LiveData case